modules/__tests__/_bc-Link-test.js

```diff
@@ -52,7 +52,7 @@ describe('v1 Link', function () {
               Michael
             </Link>
             <Link
-              to="hello/ryan" query={{ the: 'query' }}
+              to="/hello/ryan" query={{ the: 'query' }}
```

Comment:
To elaborate on "not supposed to work" – links like this will do random, dangerous things when used with e.g. browser history right now, given the lack of support there for relative links.